kedro/framework/cli/starters.py

```diff
@@ -299,6 +299,13 @@ def new(  # noqa: PLR0913
 
     _create_project(project_template, cookiecutter_args)
 
+    if prompts_required and not config_path:
```

Comment:
Is it true that it will be shown when using with starter? Where no tools allowed?

Reply:
Addressed in https://github.com/kedro-org/kedro/pull/3439/commits/8f86cf86ee49948e79a9bf2afe8abd2d58543a5f